samples/bookinfo/routing/certmanager-egress-rules.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Shouldn't this be in the chart ? Bookinfo is a sample, if you need this for certmanager needs to be next to it.
I believe it is excluded by default.